### PR TITLE
Update batch and queues page

### PR DESCRIPTION
**Pull request by dimitriskr**
Update the part of running batches through Drush command with new code for Drush 12. Change the order of other parts

### CODE REVIEW

File: book/bq.md [hunk 1]
```diff
@@ -388,39 +388,89 @@ You follow all the same procedures as shown in the form or controller style abov
...
+    $batch->setFinishCallback([ExampleCommands::class, 'batchFinishImportData']);
+    batch_set($batch->toArray());
+    $this->logger()->notice("Batch is set.");
+    $this->logger()->notice("Start the batch process.");
+    drush_backend_batch_process();
+    return DrushCommands::EXIT_SUCCESS;
+  }
+  public static function batchImportData($data) {
+    // The structure of $data is not valid for $node_manager->create($data), it's only for showcase.
+    $node_manager = \Drupal::entityTypeManager()->getStorage('node');
```
Comment (selwynpolit) [inline book/bq.md:458]: @dimitriskr I like this example.  Do you think we should tweak it a little to make it create some article nodes (for example) ?

Comment (dimitriskr) [reply book/bq.md:458]: Sure. Any particular structure with fields or a general one?

Comment (selwynpolit) [reply book/bq.md:458]: Great.  How about just fill in a title, body and author in an article node?  I just try to always have actual usable examples in the code if possible,

Comment (selwynpolit) [reply book/bq.md:458]: @dimitriskr I hope I didn't scare you off with my suggestion

Comment (dimitriskr) [reply book/bq.md:458]: @selwynpolit Haha no no, life happens. I'll get to it soon

Comment (dimitriskr) [reply book/bq.md:484]: Is it better now? @selwynpolit 

(better late than never)

Comment (selwynpolit) [reply book/bq.md:484]: looks great @dimitriskr Thanks!

**Review by selwynpolit — APPROVED** [review]

**Pull request merged**